handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -93,6 +93,8 @@
         }
     }
 
+    final boolean startTls;
```

Comment:
I need it to be package-visible to access it in SniHandler to call the static method to create an SslHandler from an SslEngine.
Another solution was to delegate the whole SslHandler (engine included) to the context in SniHandler, but then we can't do the try/finally in case of problem.
Or maybe we should do this try/finally in SslContext.newHandler in any case?